stable/openldap/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: openldap
 home: https://www.openldap.org
-version: 0.2.4
+version: 0.2.5
```

Comment:
can you please bump to 0.3.0 since this is not a bug fix?

Reply:
yes, sure, thanks for that